.teamcity/src/main/kotlin/configurations/DocsTest.kt

```diff
@@ -86,7 +86,7 @@ class DocsTest(
     applyTestDefaults(
         model,
         this,
-        "docs:quickTest docs:platformTest",
+        "docs:docsTest${if (testSplitType == EXCLUDE) " docs:checkSamples" else ""}",
```

Comment:
❔ Would it make sense to run `docs:docsTest docs:checkSamples` explicitly to not rely on the dependency `checkSamples` -> `docsTest`?

Reply:
This is running them explicitly: run `docs:docsTest` in bucket 1/2/3; run `docs:docsTest docs:checkSamples` in bucket 4.